crypto/nrf_cc310_mbedcrypto/include/mbedtls/platform_alt.h

```diff
@@ -87,12 +77,21 @@ typedef struct mbedtls_rng_workbuf_internal {
  *
  */
 typedef struct {
-    mbedtls_rng_context_internal    rnd_context;
```

Comment:
Why was the rng_context removed, couldn't find any mention of it in the commit message.

Reply:
This is simply because the `platform_alt.h` in
` ./nrf_security/include/mbedcrypto_glue/mbedtls/platform_alt.h`
has been aligned with the one found in
`./crypto/nrf_cc310_mbedcrypto/include/mbedtls/platform_alt.h`

Reply:
ah okei :+1:  tnx for the answer.